apps/snitch_core/lib/core/data/model/zone/country_zone.ex

```diff
@@ -0,0 +1,131 @@
+defmodule Snitch.Data.Model.CountryZone do
+  @moduledoc """
+  CountryZone API
+  """
+  use Snitch.Data.Model
+
+  import Ecto.Query
+
+  alias Ecto.Multi
+  alias Snitch.Data.Schema.{CountryZoneMember, Zone, Country}
+
+  @doc """
+  Creates a new state `Zone` whose members are `country_ids`.
+
+  `country_ids` is a list of primary keys of the `Snitch.Data.Schema.Country`s that
+  make up this zone. Duplicate IDs are ignored.
+  """
+  @spec create(String.t(), String.t(), [non_neg_integer]) :: term
```

Comment:
The return type is very lengthy to type so I skipped it. The `Ecto.Multi` specs also skip it :grin: